moderation/templates/moderation/moderate_object.html

```diff
@@ -1,43 +1,5 @@
 {% extends "admin/change_form.html" %}
 {% load i18n %}
-{% block extrahead %}
-<style type="text/css">
```

Comment:
Why are you removing this ? as far as I remember this was need to display properly the image fields. Does it work on image fields without it ?

Reply:
Okay, I'll undo this, but add a {{ block.super }} to this section to get other extraheads (e.g. from django-cms).